frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "Instead update 'web_root' in Kaili config map in OpenShift console."
   ],
   "homepage": "./",
+  "proxy": "http://192.168.2.140:20001/kiali",
```

Comment:
You need to remove this line

Reply:
OK, I will remove this line when I organize it later.